index.js

```diff
@@ -4,15 +4,46 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+header.style.color = "red"
 
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector("#profile")
+const name = profile.children[1];
+name.innerHTML = traveler.name;
+const nickName = profile.children[2];
+nickName.innerHTML = traveler.nickname;
+const pfp = profile.children[0];
+pfp.src= traveler.photo
+pfp.alt= traveler.name
 
 
 /***** Deliverable 4 *****/
+const animals = document.querySelector("#animals")
+const animalSightings = traveler.animalSightings
+for(let i = 0; i <traveler.animalSightings.length; i++){
+    const item = document.createElement('li');
+    item.setAttribute("data-id", `${animalSightings[i].id}`)
+    const descrip = document.createElement(`p`)
+    const descripText = document.createTextNode(`${animalSightings[i].description}`);
+    descrip.append(descripText)
```

Comment:
Nice use of createTextNode
It looks like you may have just done this to test out createTextNode.